views/Annotations/AnnotationPointWithLine.svelte

```diff
@@ -0,0 +1,35 @@
+<script>
+  import AnnotationPoint from "./AnnotationPoint.svelte";
+  import { round } from "../helpers/data.js";
+
+  // General props
```

Comment:
I think the comments are not necessary, since it's clear that those are general and the other ones for the line, due to the naming